.github/workflows/foundry-gas-diff.yml

```diff
@@ -0,0 +1,61 @@
+name: Report gas diff
+
+env:
+  RPC_URL_MAINNET: ${{ secrets.RPC_URL_MAINNET }}
+  RPC_URL_GOERLI: ${{ secrets.RPC_URL_GOERLI }}
+  RPC_URL_OPTIMISM: ${{ secrets.RPC_URL_OPTIMISM }}
+  RPC_URL_ARBITRUM: ${{ secrets.RPC_URL_ARBITRUM }}
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    # Optionally configure to run only for changes in specific files. For example:
+    # paths:
+    # - src/**
+    # - test/**
+    # - foundry.toml
+    # - remappings.txt
+    # - .github/workflows/foundry-gas-diff.yml
+
+jobs:
+  compare_gas_reports:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: 'core'
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+
+      - name: Install Foundry
+        uses: onbjerg/foundry-toolchain@v1
```

Comment:
Why this fork instead of `foundry-rs/foundry-toolchain@v1`

Reply:
I don't know. I just used the one they use.

Reply:
It looks like they are the same thing

Reply:
Probably use the new one that it redirects to then